src/main/java/pro/sky/calculator/CalculatorController.java

```diff
@@ -0,0 +1,72 @@
+package pro.sky.calculator;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/calculator")
+public class CalculatorController {
+
+    private final CalculatorServiceImpl calculatorServiceImpl;
```

Comment:
```suggestion
    private final CalculatorService calculatorServiceImpl;
```
всегда надо использовать интерфейсную ссылку